schema.go

```diff
@@ -82,8 +82,11 @@ type Schema struct {
 // and the data will not be written into the parquet file(s).
 // Note that a field with name "-" can still be generated using the tag "-,".
 //
-// Map key configuration can be pass using the `parquet-key` tag. This tag will follow the
-// same properties than the `parquet` tag.
+// The configuration of Parquet maps are done via two tags:
+//   - The `parquet-key` tag allows to configure the key of a map.
+//   - The `parquet-value` tag allows to configure the values of a map.
+//
+// When configuring a Parquet map, the `parquet` tag will configure map itself.
```

Comment:
```suggestion
// When configuring a Parquet map, the `parquet` tag will configure the map itself.
```

Reply:
The tests reference e.g. `timestamp(microsecond)` - maybe we can provide a list of valid values that users can provide, or more examples?

Reply:
You mean for the precision of the timestmap, It is there already https://github.com/segmentio/parquet-go/blob/ffd0eebbd2b0668e5e2ce0834a649dca7ae74fcc/schema.go#L65